app/models/manageiq/providers/workflows/automation_manager/workflow_instance.rb

```diff
@@ -87,6 +90,6 @@ def run(args = {})
       object.after_ae_delivery(ae_result)
     end
 
-    run_queue(:zone => zone, :role => role, :object => object) unless wf.end?
+    run_queue(:zone => zone, :role => role, :object => object, :deliver_on => 10.seconds.from_now.utc, :server_guid => MiqServer.my_server.guid) unless wf.end?
```

Comment:
@agrare Is this what you mentioned to me?

So my question here is why we need the pinning and deliver time at all?

Reply:
Because this re-queue while the docker/podman/k8s task is still running so we don't want it to spin (hence deliver on) and we need to run it on the same server so `podman inspect` will be able to see the running container.

Reply:
ohhh I see - ok, this is what I wasn't understanding in our discussion yesterday because I thought it worked differently.